pom.xml

```
@@ -68,7 +68,7 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
     <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
     <url>http://github.com/${gitHubRepo}</url>
```

I realize this issue existed before, but it would be nice to resolve it while we're adapting this plugin for Java 21.
```suggestion
    <url>https://github.com/${gitHubRepo}</url>
```